docs/changelog.rst

```diff
@@ -26,6 +26,7 @@ In Progress (unreleased)
 * :gh:issue:`1116` :mod:`ansible_mitogen`: Support for templated variable
   `ansible_ssh_user`.
 * :gh:issue:`978` :mod:`ansible_mitogen`: Support templated Ansible SSH port.
+* :gh:issue:`1083` fix: Jinja expressions are not evaluated in (not only) credential variables.
```

Comment:
This PR tries to get one templated option (`'become_user'`) working. I don't think that's sufficient to call issue #1083 fixed, since that specifies multiple options.